api/src/main/java/org/openmrs/api/OrderContext.java

```diff
@@ -65,7 +65,6 @@ public void setCareSetting(CareSetting careSetting) {
 	public Map<String, Object> getContextAttributes() {
 		if (contextAttributes == null) {
 			contextAttributes = new HashMap<>();
```

Comment:
Is this related to the changes you have made in Concept.java?

Reply:
No. This part of code was partially fixed with #2750 (by splitting up assigment and return statement) but it left us with doubled `return` statement, as you can see, so I've decided to remove a duplicate (this change really adresses the same type of a warning).

Reply:
I've addressed this issue with #2729 way before but my PR was not merged. Later another student proposed a fix to the same issue, leaving an additional, unecessary statement, which I removed now.

Reply:
For unrelated changes, we recommend separate pull requests.